indico_toolkit/association/positioning.py

```diff
@@ -42,6 +42,45 @@ def positioned_above(
             is_above = False
         return is_above
 
+    def positioned_above_overlap(
+            self, above_pos: dict, below_pos: dict, must_be_same_page: bool = True, min_overlap_percent: float = None
+        ) -> bool:
+        """
+        Check if the location of one box is above another and if the lower box's overlap is at least the given percentage.
+        Args:
+            above_pos (dict): the position expected to be above
+            below_pos (dict): the position expected to be below
+            must_be_same_page (bool, optional): required to be on same page. Defaults to True.
+            min_overlap_percent (float, optional): the minimum amount of overlap needed. Defaults to None.
+
+        Returns:
+            bool: True is above_pos is above below_pos and below_pos' amount of overlap is at least min_overlap_percent
+        """
+        if below_pos["page_num"] < above_pos["page_num"]:
```

Comment:
I think you can just raise an exception if they aren't on the same page right away and then avoid that check everywhere (and also get rid of it as a kwarg)- because look at get_min_horizontal distance function, require that the two positions be on the same page

Reply:
this should be "!=", per above we need them to be on the same page